src/common/ios/xcodeproj.ts

```diff
@@ -1,13 +1,16 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for details.
 
+import * as path from "path";
 import * as Q from "q";
 
-import {ErrorHelper} from "../../common/error/errorHelper";
-import {Log} from "../../common/log/log";
 import {FileSystem} from "../../common/node/fileSystem";
 
-import {TelemetryHelper} from "../../common/telemetryHelper";
+export interface IXcodeProjFile {
+    filename: string;
+    filetype: string;
+    projectName: string;

```

Comment:
Why does projectName use camelCasing but neither filename nor filetype do? Not a big thing, just looks a little strange :)